app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt

```diff
@@ -371,11 +372,18 @@ internal fun getFakePocketStories(limit: Int = 1): List<PocketStory> {
                 )
                 false -> add(
                     PocketSponsoredStory(
+                        id = index,
                         title = "This is a ${"very ".repeat(index)} long title",
                         url = "https://sponsored-story$index.com",
                         imageUrl = "",
                         sponsor = "Mozilla",
-                        shim = PocketSponsoredStoryShim("", "")
+                        shim = PocketSponsoredStoryShim("", ""),
+                        priority = index,
+                        caps = PocketSponsoredStoryCaps(
+                            flightCount = index,
+                            flightPeriod = index * 2,
+                            lifetimeCount = index * 3,
+                        )
```

Comment:
@Mugurell 👋🏽 
I added this to address the AC breaking change, I took it from https://github.com/mozilla-mobile/fenix/pull/25290. I'm not sure how close it's from landing as there some still discussions on the PR. Is this approach OK ?  I noticed `getFakePocketStories` is only being used in the preview.

Reply:
Looks good.
Was waiting for the AC bump to merge.
Seems like we can merge this and I'll land immediately after.
Thank you for the new AC version and a swifter merge process! 😊

Reply:
Seems like some unit tests would also need to be updated.
As discussed I'll borrow the AC version bump and land first to address all breaking changes.